drone-fly-app/src/main/java/com/expediagroup/dataplatform/dronefly/app/service/DroneFlyNotificationService.java

```diff
@@ -56,8 +56,11 @@ public void notifyListeners() throws DroneFlyException {
       ApiaryListenerEvent event = reader.read();
       ListenerEvent hiveEvent = converterService.toHiveEvent(event);
       List<MetaStoreEventListener> listeners = listenerCatalog.getListeners();
-      log.info("Read event: {}", event.getEventType().toString());
-      log.info("Listeners being notified: {}", listeners.size());
+      log.info("Notifying event type: {}", event.getEventType().toString());
+      log.debug("DB Name: {}", event.getDatabaseName().toString());
```

Comment:
Maybe put DB name and Table name on one line in the logs?

Reply:
Done. Changed to `Qualified Table Name: {}.{}`